core/src/components/content/test/scroll-snap/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Content - Scroll Snap</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link href="../../../../../css/ionic.bundle.css" rel="stylesheet">
+  <link href="../../../../../scripts/testing/styles.css" rel="stylesheet">
+  <script src="../../../../../scripts/testing/scripts.js"></script>
+  <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+  <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script></head>
+
+<body>
+  <ion-app>
+    <ion-header translucent>
+      <ion-toolbar>
+        <ion-title>Content - Scroll Snap</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content fullscreen>
+      <ion-header collapse="condense">
+        <ion-toolbar>
+          <ion-title size="large">Content - Scroll Snap</ion-title>
+        </ion-toolbar>
+      </ion-header>
+      <!-- It is required to group up the content -->
+      <div class="ion-padding scroll-snap-content">
```

Comment:
Will developers need to add the "scroll-snap-content" class?

Reply:
> All the content (except ion-header) should be wrapped in a container (scroll-snap-content), otherwise each children will snap.

Just saw this in the original post. Is there a way we can do this without adding the class? The collapsible large title already takes a bit of work to wire up, so I would like to avoid adding more to that.

Maybe we can target only by `ion-header[collapse="condense"]` in the CSS?

Reply:
The CSS class is not required. I can move to a generic selector `::slotted(*)`, but a wrapper container is still required.

> Maybe we can target only by ion-header[collapse="condense"] in the CSS?

That could be controlled by `'scroll-snap': ionHeaderCondenseExists && isPlatform('ios')` but not resolves the needed of a wrapper.
`<ion-content>` children need `scroll-snap-align: start` since its default value is `scroll-snap-align: none` and it does not work as we want.